web/controllers/user_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule MovieWagerApi.UserController do
+  use MovieWagerApi.Web, :controller
+
+  alias MovieWagerApi.{Repo, User, UserSerializer}
+
+  def show(conn, %{"id" => identifier}) do
```

Comment:
Now that I think of it, @SViccari do you think a more elegant solution would be to make the `username` the primary key for this table?